CHANGELOG.rst

```diff
@@ -7,14 +7,28 @@ These are notable changes in XBlock.
 Unreleased
 ----------
 
+1.8.2 - Unreleased
```

Comment:
The version should also be bumped in `xblock/__init__.py`, also given the backwards incompatible change to the `__init__` method signature for the `Runtime` class, shouldn't this be a major version bump?

Reply:
I think this question is not resolved, given the removal, and signature change, this should be a major version bump.

Reply:
I've already updated the version to `2.0.0` and also bumped in `xblock/__init__.py`.

https://github.com/openedx/XBlock/pull/680/files#diff-82a12ef53486e3e312a8133bf8a14c5d6be3983ed0d037ea623ca4582d3f04ec

https://github.com/openedx/XBlock/pull/680/files#diff-2c623f3c6a917be56c59d43279244996836262cb1e12d9d0786c9c49eef6b43c